examples/megssm/mne_util.py

```diff
@@ -0,0 +1,290 @@
+""" MNE-Python utility functions for preprocessing data and constructing
```

Comment:
Move all of these files from `examples` into `mne_connectivity/state_space/*.py` for example